app/src/main/java/com/kickstarter/ui/extensions/ImageViewExt.kt

```diff
@@ -21,17 +21,9 @@ import com.bumptech.glide.request.RequestOptions
 import com.bumptech.glide.request.target.Target
 import com.google.firebase.crashlytics.FirebaseCrashlytics
 import com.kickstarter.R
-import com.kickstarter.libs.utils.WebUtils
-import com.kickstarter.libs.utils.extensions.getEnvironment
+import com.kickstarter.libs.utils.extensions.header
 import com.kickstarter.libs.utils.extensions.isKSApplication
-
-const val header = "User-Agent"
-const val ua = "Custom-UA"
-
-fun userAgent(context: Context) = if (context.applicationContext.isKSApplication()) {
```

Comment:
Converted into a context extension function, to be re-used for VideoPlayer as well